packages/legacy-browser-alert/src/index.tsx

```diff
@@ -0,0 +1,84 @@
+import * as React from 'react';
+
+const ieRegexes = [
+    /Trident\/7\.0.*rv\:([0-9\.]+).*\).*Gecko$/,
+    /MSIE\s([0-9\.]+);.*Trident\/[4-7].0/,
+    /MSIE\s(7\.0)/
+];
+
+type Props = {
+  locale?: 'ko' | 'en';
+  sx?: React.CSSProperties;
+  onClose?: () => void;
+}
+
+const AlertComponent = ({ locale = 'ko', sx, onClose }: Props) => {
+  const userAgent = window?.navigator?.userAgent;
+  const isIE = ieRegexes.some(regex => regex.test(userAgent));
+  if (!isIE) {
+    return null;
+  }
+
+  const containerStyle = {
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'center',
+    width: '100%',
+    padding: '1rem 0',
+
+    fontSize: '1rem',
+    textAlign: 'center' as const,
+
+    backgroundColor: '#FFF0F0',
+    color: '#D22030',
+    ...sx,
+  };
+
+  const anchorStyle = {
+    color: '#D22030',
+    fontWeight: 700,
+  }
+
+  const spanStyle = {
+    marginLeft: '0.25rem',
+  }
```

Comment:
```suggestion
  const spanStyle = {
    marginLeft: '0.25rem',
    flex: 1,
  }
```
브라우저 사이즈 쪼그라들었을 때 아이콘과 닫기버튼이 쪼그라듭니다! 이 부분에 flex: 1 적용하니까 그대로 유지되어서 추가해주면 좋을것 같습니다